### PR TITLE
Lesser drones now die upon ghosting

### PR DESCRIPTION
**Pull request by Birdtalon**
# About the pull request

https://forum.cm-ss13.com/t/ghosted-lesser-drones-should-just-die/4356/10

Lessers now die upon ghosting and are not offered to observers.

# Explain why it's good for the game

Nobody likes trying to join as xeno or see their position in the queue only to see

Lesser Drone (420)
Lesser Drone (69)

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
del: Lesser drones die upon ghosting and are not offered to observers.
/:cl:

### CODE REVIEW

**Comment by fira** [comment]
Might make sense to not gib them if aghosting

**Comment by Birdtalon** [comment]
> Might make sense to not gib them if aghosting

But where's the fun in that!?

**Comment by Drulikar** [comment]
If its merely because of the queue, you have the queue message in the status panel. Clicking the join as xeno button does not force an update of your queue (unless you don't have any cached message at all).

**Comment by ihatethisengine** [comment]
Do the same for facehuggers

**Comment by Drulikar** [comment]
Will note they already poof after 7 mins #3886

File: code/modules/mob/living/carbon/xenomorph/castes/lesser_drone.dm [hunk 1]
```diff
@@ -95,3 +95,11 @@
 	if (PF)
 		PF.flags_pass = PASS_MOB_IS_XENO|PASS_MOB_THRU_XENO
 		PF.flags_can_pass_all = PASS_MOB_IS_XENO|PASS_MOB_THRU_XENO
+
+/mob/living/carbon/xenomorph/lesser_drone/ghostize(can_reenter_corpse = FALSE, aghosted = FALSE)
+	. = ..()
+	if(. && !aghosted)
+		src.gib()
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/castes/lesser_drone.dm:102]:
```suggestion
		gib()
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**